coral-common/src/main/java/com/linkedin/coral/common/HiveViewTable.java

```diff
@@ -156,8 +156,7 @@ private static boolean isFieldListCastRequired(List<RelDataTypeField> inputField
 
     int inputIndex = 0;
     int inputFieldsSize = inputFields.size();
-    for (int i = 0; i < castToFields.size(); i++) {
-      RelDataTypeField castToField = castToFields.get(i);
+    for (RelDataTypeField castToField : castToFields) {
```

Comment:
Are you using any tool to catch those code?

Reply:
IntelliJ code analyzer is a good helper 😆